src/Resources/app/storefront/src/snippets/de_DE/unzer-payment.de-DE.json

```diff
@@ -64,6 +64,6 @@
             "title": "Ihre gespeicherten Zahlungsmittel"
         },
         "finishPaymentFailed": "<a href=\"%editOrderUrl%\">Zahlung ändern</a>",
-        "birthdayInvalid": "Geburtsdatum ungültig"
+        "birthdayInvalid": "Du musst mindestens 18 Jahre alt sein."
```

Comment:
Are messages in Shopware informal in general ('Du' instead of 'Sie')?

Reply:
Updated the message to a formal version based on other messages in the plugin 'Wählen Sie...', 'Ihre gespeicherten Zahlungsmittel'